### PR TITLE
fix($theme-default): fix wrong editLink (close: #1115, #1125)

### PR DESCRIPTION
**Pull request by Shigma**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

#1115, #1125

### CODE REVIEW

**Comment by Shigma** [comment]
cc @meteorlxy

File: packages/@vuepress/theme-default/components/Page.vue [hunk 1]
```diff
@@ -146,8 +140,8 @@ export default {
         return (
           base.replace(endingSlashRE, '')
            + `/src`
-           + `/${docsBranch}`
-           + (docsDir ? '/' + docsDir.replace(endingSlashRE, '') : '')
+           + `/${docsBranch}/`
+           + (docsDir ? docsDir.replace(endingSlashRE, '') + '/' : '')
```
Comment (ulivz) [inline packages/@vuepress/theme-default/components/Page.vue]: It seems that changes here is unnecessary?

Comment (Shigma) [reply packages/@vuepress/theme-default/components/Page.vue]: In past implementations, `path` is prefixed with a `/`, however in the pr, `path` is not prefixed with `/`. So adding a ending slash is a must.

**Comment by ulivz** [comment]
Here could be a typo, while this issue addressed #1425 instead of #1125

**Pull request merged**